test/liquidity-curve.test.js

```diff
@@ -133,12 +133,14 @@ describe('LiquidityCurve', function () {
     })
 
     it('cleans up duplicate points', function () {
-      const curve1 = new LiquidityCurve([ [1, 0], [50000000001, 49800199999], [100000000000001, 49800199999] ])
-      const curve2 = new LiquidityCurve([ [2, 0], [50000000001, 49800199999], [100000000000001, 49800199999] ])
+      const curve1 = new LiquidityCurve([ [0, 0], [2, 2] ])
+      const curve2 = new LiquidityCurve([ [1, 0], [2, 9] ])
```

Comment:
why not keep the test with the real-world values? The unit of these curves is typically nano-dollars or nano-euros, so values < 10000000 are not so realistic